homeassistant/exceptions.py

```diff
@@ -13,6 +13,8 @@
 class HomeAssistantError(Exception):
     """General Home Assistant exception occurred."""
 
+    _translate: bool = False
+
```

Comment:
I think we should store the translated message instead
```suggestion
    _message: str | None = None
```